sound/soc/sof/ops.h

```diff
@@ -193,6 +193,14 @@ static inline int snd_sof_dsp_cmd_done(struct snd_sof_dev *sdev)
 		return 0;
 }
 
+static inline int snd_sof_dsp_reply_done(struct snd_sof_dev *sdev)
```

Comment:
What is th ediffernce between reply_done() and cmd_done() ? Cant they be merged into one function ?